README.md

```diff
@@ -54,6 +54,44 @@ FAILED (errors=4)
 
 ## Release
 
+### Tag branches
+
+Tag repositories with their respective next tag.
+
+- branches without any new commits since the last tag will not be re-tagged.
+- `master` branches will be tagged with the next `{minor}` and `{patch}` of `0`.
+  For example, a repo with `master` tagged `1.2.0` will be tagged `1.3.0`.
+- all other branches (e.g: `release-*`) will be tagged with the next `{patch}`.
+  For example, a repo with `release-1.27.x` tagged `1.2.3` will be tagged `1.2.4`.
+
+At time of writing, tagging designated (`master` and `release-*`) branches will:
+
+1. (on `master`) provide a commit SHA to branch at for a new Spinnaker Release,
+   e.g: `1.28.0`.
```

Comment:
I'm not sure 1.28.0 is a great example here....since typically microservice versions don't line up with overall spinnaker versions.

Reply:
Thanks, have reworded, how does this sound?  https://github.com/spinnaker/buildtool/pull/190/commits/2ced9a75c5a496beb357387de79b1596ecb7cc5c#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R69

> 1. (on `master`) provide a commit SHA to create new `release-*` branches at.

Reply:
Yeah that works.  The grammar weenie in me bristles a little with `at` ending a sentence, but
```
(on master) provide a commit SHA with which to create a new `release-*` branch
```
is a bit of a mouthful.

I do see another potential place to improve...the text before this says
```
At time of writing, tagging designated (`master` and `release-*`) branches will:
```
So that's talking about more than one branch (`branches`)....but here we're talking about one commit SHA and so I figure that's also one branch....

But maybe it's better if it all talks about one branch?